esp/src/src/Session.ts

```diff
@@ -45,7 +45,10 @@ getBuildInfo().then(info => {
     dojoConfig.currencyCode = info["currencyCode"] ?? "";
 });
 
-export const formatCost = d3Format(".2f");
+export function formatCost(value) {
```

Comment:
value should be typed as 
```ts
export function formatCost(value?:number|string):string {...}
```
And move the edge case handling into the function (empty string, undefined etc.).

Reply:
I thought for falsy values it should return an empty string. And I added that logger line, but that might not be useful?